src/components/Form/DemarcheSelectionNotification.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import Loader from '../atoms/Loader';
+import { isEmpty } from 'lodash';
+import OpenInNewIcon from '../icons/open-in-new';
+
+const DemarcheSelectionNotification = ({
```

Comment:
Not sure if this is convention but I put this notification component next to CopyNotification and HasNextNotification